src/osd/OSD.cc

```diff
@@ -7134,21 +7134,24 @@ void OSD::_committed_osd_maps(epoch_t first, epoch_t last, MOSDMap *m)
 	if (r != 0) {
 	  do_shutdown = true;  // FIXME: do_restart?
           network_error = true;
-          dout(0) << __func__ << " marked down: rebind failed" << dendl;
+          dout(0) << __func__ 
```

Comment:
It looks better if you do all the changes the same way and seems the preferred option is to be `<<` aligned here:
```


dout(0) << __func__ << " marked down:"
        << " marked down: rebind cluster_messenger failed" << dendl;
```